packages/react-storage/src/components/StorageBrowser/composables/OverwriteToggle.tsx

```diff
@@ -25,6 +25,7 @@ export const OverwriteToggle = ({
       id={OVERWRITE_TOGGLE_ID}
       onChange={onToggle}
       type="checkbox"
+      aria-label="overwrite-toggle"
```

Comment:
Align with others i.e. `Overwrite Toggle`

Reply:
Updated approach to find the overwrite checkbox. 

Looks like 
- managedAuth is using the default OverwriteToggle composable. This contains input checkBox
- defaultAuth is overriding default composable with it's own. It's using label + span + icon instead.

Updated to a more generic approach, "look for certain text inside a label"

Reply:
rebased ignore comment